dlrover/python/master/hyperparams/simple_strategy_generator.py

```diff
@@ -118,15 +125,36 @@ def _generate_dataloader_config(
             return DataLoaderConfig(
                 updated_version,
                 dataloader_config.dataloader_name,
+                batch_size,
                 updated_batch_size,
                 0,
                 0,
             )
-        else:
-            return dataloader_config
 
-    def _generate_optimizer_config(self):
-        return OptimizerConfig(5, 6)
+    def _generate_optimizer_config(self, dataloader_config, optimizer_config):
+        batch_size = dataloader_config.batch_size
+        last_batch_size = dataloader_config.last_batch_size
+
+        try:
+            ratio = batch_size / last_batch_size
+        except ZeroDivisionError:
+            ratio = 1
+        coefficient = math.sqrt(ratio)
+
+        update_version = optimizer_config.version + 1
+        update_learning_rate = optimizer_config.learning_rate * coefficient
```

Comment:
write down the math formual here to illustrate this procedure. 
// learn-rate = learn-rate * xxx